include/baresip.h

```diff
@@ -1686,7 +1686,7 @@ int sip_req_send(struct ua *ua, const char *method, const char *uri,
 #endif
 
 
-int  module_preload(const char *module);
+int  module_preload(const char *module, const struct conf *cfg);
```

Comment:
perhaps the config can be fetched inside this function, so that we dont have to break the API ?

Reply:
Ah, sorry - didn't catch that `module_preload` was part of the public API. Will follow up. 

BTW, any interest in patches to add a C API that avoids global variables? (ie, by explicitly passing a "context" object around)

Reply:
Addressed with commit `bbcb50cd`.